CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 ## To Be Released
 
-* v2.0.0
+* feat(network): add missing fields to network and IPAM `CapabilitiesResponse`
```

Comment:
This is a missing field from https://github.com/Scalingo/go-plugins-helpers/pull/63